public-ui/src/environments/test-env-base.ts

```diff
@@ -0,0 +1,7 @@
+// The values are shared across the deployed test env as well as the local dev
+// environments.
+export const testEnvironmentBase = {
+  allOfUsApiUrl: 'https://api-dot-all-of-us-workbench-test.appspot.com',
```

Comment:
Nit: you probably don't need the allOfUsApiUrl in public-ui at this point?

Reply:
Removed.